.github/workflows/workflow_test.yml

```diff
@@ -44,6 +44,14 @@ jobs:
         mode: lint
         workflows: true
         repository-list: "${{ needs.setup.outputs.repository-list }}"
+    - name: IWC specific lint
+      run: |
+        for REPO in ${{ needs.setup.outputs.repository-list }}; do
+          echo $REPO
+          cd $REPO
+          python3 scripts/lint_repo.py
```

Comment:
This is not working because the path `scripts/lint_repo.py` is not working anymore once you have done `cd` but I do not know where can I find the variable that contains the path where the github repo has been checked out.